README.md

```diff
@@ -19,8 +19,6 @@
 
 # Apache Arrow
 
-[![Build Status](https://ci.appveyor.com/api/projects/status/github/apache/arrow/branch/master?svg=true)](https://ci.appveyor.com/project/ApacheSoftwareFoundation/arrow/branch/master)
-[![Fuzzing Status](https://oss-fuzz-build-logs.storage.googleapis.com/badges/arrow.svg)](https://bugs.chromium.org/p/oss-fuzz/issues/list?sort=-opened&can=1&q=proj:arrow)
```

Comment:
We sill use OSS-Fuzz.
Could you keep this badge?

Reply:
I made a new pull request: https://github.com/apache/arrow/pull/10280. 

Is oss-fuzz being used for packages besides C++?

Reply:
Yes.

Reply:
Great. Then I'll close this pull request.